src/index.ts

```
@@ -117,15 +152,15 @@ export class PatchyInternetQImpl {
 		
 		try {
 			await this.process(this.queue.head);
-			this.dequeue();
+			await this.dequeue();
 		} catch (err) {
 			throw err;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)